modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -61,25 +61,37 @@ EOF
 ----
```

Comment:
can't comment on it, but line 38:
```
Tokens: <none>
```
Can you update to a previous value:
`Tokens: thanos-token-c422q`

It's a nit pick, but in 4.15 and below, by default, clusters will indeed generate the Token, so I think it makes sense to align the output with the default / most likely output of a cluster.

I think it also helps customers see what the token output should look like, given the next step is optional. Might help them realize "oh mine says `<none>`, but the example shows a token, maybe I need to do this optional step to get one".

Reply:
:heavy_check_mark: